elfpy/utils/apeworx_integrations.py

```diff
@@ -209,6 +212,122 @@ def get_pool_state(tx_receipt: ReceiptAPI, hyperdrive_contract: ContractInstance
     return pool_state
 
 
+PoolInfo = namedtuple("PoolInfo", ["start_time", "block_time", "term_length", "market_state"])
+
+
+# TODO: remove commenets after verifying the accuracy of this function through more testing
```

Comment:
Can you remove these now? If not can you make an issue for this TODO?

Reply:
added #423